doc/contributing/issues.md

```diff
@@ -60,5 +60,12 @@ activities, such as applying labels and closing/reopening/assigning issues.
 For more information on the roles and permissions, see ["Permission levels for
 repositories owned by an organization"](https://docs.github.com/en/github/setting-up-and-managing-organizations-and-teams/repository-permission-levels-for-an-organization#permission-levels-for-repositories-owned-by-an-organization).
 
+triagers are requested to abstain from below actions:
+* approve or request changes to a pull request
+* edit wikis
+
+triagers are requested to apply caution while performing below actions:
+* close or assign issues or prs
```

Comment:
This should also apply to collaborators, and it's arguably very vague (to a point I'm not sure it's adding much)

Reply:
pls suggest, if you have a better wording. my intent is: in the absence of a clear direction here, triagers  could act on the stated objective above (to reduce backlog) alone, without worrying about untimely close actions that can potentially discourage the OP?

Reply:
```suggestion
When triagging issues and PR:

* Show patience and empathy, especially to first-time contributors.
* Show no patience towards spam or troll, close the issue without interacting with it and
  report the user to the moderation repository.
* If you're not able to reproduce an issue, leave a comment asking for more info and
  add the `needs more info` label.
* Ideally issues should be closed only when they have been fixed or answered. Closing
  an issue earlier can be seen as dismissive from the point of view of the reporter.
  Always try to communicate the reason for closing the issue.
```